core/src/main/resources/reference.conf

```diff
@@ -85,6 +85,13 @@ system {
 
   # Default number of cache read workers
   number-of-cache-read-workers = 25
+  
+  io {
+    # Global Throttling - This is mostly useful for GCS and can be adjusted to match
+    # the quota availble on the GCS API
+    number-of-requests = 100000
+    per = 100 seconds
```

Comment:
Just noting that `per 100 seconds` is specific to GCS as far as I know though (granted that currently the only cloud filesystem we support is GCS).
I don't mind switching it to `queries-per-100-seconds` but it will probably need to change again when we support other cloud based FSs